helm/linkerd2-cni-app/Chart.yaml

```diff
@@ -20,4 +20,6 @@ name: linkerd2-cni-app
 sources:
 - https://github.com/linkerd/linkerd2
 - https://github.com/giantswarm/linkerd2-cni-app
-version: [[ .Version ]]
+version: 0.3.1-beta-fc0798077fdec63274b019706e9c08c861b82d98
```

Comment:
Should it be that version? In general releasing a `beta` would be fine for me.

Reply:
oh no, good catch that is a leftover from failed `abs` runs